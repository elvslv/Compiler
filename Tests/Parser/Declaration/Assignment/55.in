type tarr = array[1..10] of integer;
var a: array[1..10] of tarr;
b: tarr;
begin
	a[5][4] := b[1];
end.