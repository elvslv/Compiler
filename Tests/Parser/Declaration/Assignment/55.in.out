type TARR = array [const 1..const 10] of INTEGER;
var A : array [const 1..const 10] of TARR;
var B : TARR;
[]   
|    
|___A
|    
|___5
